components/atom/input/src/Input/Component/index.js

```diff
@@ -15,8 +15,18 @@ const ERROR_STATES = {
 }
 
 class Input extends Component {
-  changeHandler(ev, onChange) {
-    onChange && onChange({value: ev.target.value, ev})
+  changeHandler = ev => {
+    const {onChange} = this.props
+    const {
+      target: {value}
+    } = ev
+    onChange && onChange({value, ev})
+  }
+
+  onEnterHandler = ev => {
+    const {onEnter} = this.props
+    const {key} = ev
+    if (key === 'Enter') onEnter && onEnter(ev)
```

Comment:
you could do `if (onEnter && key === 'Enter')`

I know that sometimes the Tag is added pressing on comma, maybe the key should be a prop with Enter as default value or something like that